### PR TITLE
luci-mod-network: switch: Fix multi instances handling

### PR DESCRIPTION
**Pull request by ptpt52**
This fix issue https://github.com/openwrt/luci/issues/5714

### CODE REVIEW

File: modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js [hunk 1]
```diff
@@ -299,7 +297,11 @@ return view.extend({
 				return true;
 			};
 
+			o.port_opts = [];
+			var port_opts = o.port_opts;
```
Comment (jow-) [inline modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:301]: @ptpt52 - this line (301) seems redundant. Can you please confirm that your fix still works after removing it?

Comment (ptpt52) [reply modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:301]: no, it is not redundant.

Comment (jow-) [reply modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:301]: The the code should be refactored. Having a `var port_opts` in both the outer and inner scope is confusing.

Comment (ptpt52) [reply modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:301]: https://github.com/openwrt/luci/blob/c7cae787e18a4eea647978bb7ac6dae5fbad7d8b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js#L367
https://github.com/openwrt/luci/blob/c7cae787e18a4eea647978bb7ac6dae5fbad7d8b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js#L370

Comment (ptpt52) [reply modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:301]: If convenient you can help rename it

Comment (jow-) [reply modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:301]: Yep seen it now, could you please change it like that:
Merge line 300 & 301 into a chained assignment
```javascript
var port_opts = o.port_opts = [];
```

File: modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js [hunk 2]
```diff
@@ -299,7 +297,11 @@ return view.extend({
...
+
 			o.write = function(section_id, value) {
+				var port_opts = this.port_opts;
```
Comment (jow-) [inline modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:304]: Drop this line...

File: modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js [hunk 3]
```diff
@@ -299,7 +297,11 @@ return view.extend({
...
 				var topology = this.section.topology,
 				    values = [];
```
Comment (jow-) [inline modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:307]: ... and change all subsequent uses of `port_opts` within the `o.write()` function scope to `this.port_opts`

**Review by jow- — CHANGES_REQUESTED** [review]

File: modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js (hunk 3 above)
Comment (ptpt52) [reply modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:305]: done

Comment (jow-) [reply modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js:305]: Perfect, thank you!

**Pull request merged**